install_config/configuring_authentication.adoc

```diff
@@ -1435,14 +1435,40 @@ characters long, to select AES-128, AES-192, or AES-256.
 ====
 
 [[configuring-user-agent]]
-== User Agent
+== Preventing CLI Version Mismatch With User Agent
 
-Administrators can prevent clients from accessing the API by their `User-Agent`
-header in the new `*userAgentMatching*` configuration setting.
+{product-title} implements a user agent to prevent an application developer's
```

Comment:
Yeah I originally had what you suggest, but didn't like the construction. I've changed it though.